app/client.cjsx

```diff
@@ -36,8 +36,8 @@ Raven?.setUserContext piwik: piwik.getVisitorId()
 require('react-tap-event-plugin')()
 
 # English data added as default
-addLocaleData require "react-intl/lib/locale-data/fi"
-addLocaleData require "react-intl/lib/locale-data/sv"
+for language in config.availableLanguages
+  addLocaleData require "react-intl/lib/locale-data/" + language
```

Comment:
This will pull all existing languages into the js bundle, could we maybe add the current locale to the page on the server (from the selected locale), and fetch a new locale definition from the server when the user decides to switch?